src/twisted/topfiles/9126.feature

```diff
@@ -0,0 +1 @@
+twisted.python.url and twisted.python._url were modified to rely on hyperlink, a new package based on the Twisted URL implementation. Hyperlink adds support for IPv6 (fixing #8069), correct username/password encoding, better scheme/netloc inference, improved URL.click() behavior (fixing #8184), and more. For full docs see hyperlink.readthedocs.io and the CHANGELOG in the hyperlink GitHub repo.
```

Comment:
I don't think that this should be a .feature.

I think that this should be a .removal fragment informing that twisted.python.url is deprecated and people should use new Hyperlink package.